s2p/rectification.py

```diff
@@ -324,7 +324,7 @@ def rectify_pair(im1, im2, rpc1, rpc2, x, y, w, h, out1, out2, A=None, sift_matc
        
             
     # If there are still no matches, raise the no matches error.
-    if matches is None or len(matches) < 4:
+    if matches is None or len(matches) < 10:
```

Comment:
let me change this, it can be 4 it's just there's a certain case when if we get 1 match the len is also 4 and it breaks. Will change it to shape[0] < 4 and is 2d